javascript/javascript2/week1/main.js

```diff
@@ -0,0 +1,15 @@
+"use strict"
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+function renderProducts(products) {
+    const ul_tag = document.querySelector("ul");
+    for (const product of products){
+        const li_tag = document.createElement("li");
+        li_tag.innerHTML= "<h2>"+product.name+"</h2>"+"<span> price: "+product.price+"</span>"+"<br>"+"<span> rating: "+product.rating+"</span>"+"<br>";
```

Comment:
That works, but didn't we just learn how to create `h2`, `span` tags with the Javascript DOM API?

Reply:
Now created all the tags in js..thank you